parsing/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -2780,6 +2780,19 @@ SqlNode TableRefOrJoinClause() :
     { return e; }
 }
 
+/**
+ * Parses a prefix row operator like NOT.
+ */
+SqlPrefixOperator PrefixRowOperator() :
+{}
+{
+    <PLUS> { return SqlStdOperatorTable.UNARY_PLUS; }
+|   <MINUS> { return SqlStdOperatorTable.UNARY_MINUS; }
```

Comment:
nit: I know this matches the style in the original PrefixRowOperator, but I'd suggest making this OR style match what's predominantly used elsewhere in this file (e.g. what's used in TableRefOrJoinClause() above).

Reply:
Sure that sounds good, will update